src/test/lib/ApiTest.cpp

```diff
@@ -1735,6 +1735,406 @@ class SecConfigTestContext {
     }
 };
 
+void QuicTestGlobalSetParam()
+{
+    //
+    // QUIC_PARAM_GLOBAL_SUPPORTED_VERSIONS
+    //
+    {
+        TestScopeLogger logScope("QUIC_PARAM_GLOBAL_SUPPORTED_VERSIONS is get only");
```

Comment:
I see camelCase being used for variables called "logScope" and "dummy". The existing code style is to use PascalCase for all variable names.

Reply:
"logScope" is used by other tests though. need to fix all later